packages/example-browser/src/main.ts

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2018 TypeFox GmbH (http://www.typefox.io). All rights reserved.
  * Licensed under the MIT License. See License.txt in the project root for license information.
  * ------------------------------------------------------------------------------------------ */
-require('monaco-editor-core');
+require('monaco-editor');
```

Comment:
why are we using require instead of import here?

Reply:
This file is gone :-)